topdown/numbers.go

```diff
@@ -53,6 +54,23 @@ func builtinNumbersRange(bctx BuiltinContext, operands []*ast.Term, iter func(*a
 	return iter(ast.NewTerm(result))
 }
 
+func builtinRandIntn(a ast.Value) (ast.Value, error) {
```

Comment:
The implementation will have to be updated to ensure that multiple calls to `rand.intn` (with the same arguments) inside the same query return the same value (this is an important property of all expressions in Rego). See https://github.com/open-policy-agent/opa/blob/main/topdown/uuid.go#L14 for an example.